wordpress-mobile/gutenberg-mobile
# Fix Video block crashing on drawing on Android

Quick fixes https://github.com/wordpress-mobile/WordPress-Android/issues/9943 by making sure to not set background color on Android. 

To test:
See details https://github.com/WordPress/gutenberg/pull/15857

Update release notes:

- [ x ] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.
